### ISSUE TITLE
status of the message-pact branch

### ISSUE THREAD

**Issue by acottais** [pact-foundation/pact-net#208]
I have successfully set up Pact.Net in my project for testing HTTP interactions between microservices. I need to test also interactions from message queues (RabbitMQ in my case)

What is the status of the message-pact branch ? Is it safe to use for message contracts on both side (consumer and provider) ?

**Comment by neilcampbell** [comment]
Yeah it should be. However it's subject to change, given that it's not yet in master.
Please let us know if you hit any issues.

**Comment by Dreamescaper** [comment]
@neilcampbell 
Thanks for answer! Two things I'd like to ask (wasn't able to find in https://github.com/pact-foundation/pact-net/issues/162 discussion).
1. Is it going to merged anytime soon? Do you need any help with that?
2. Would it be possible to have some preview nuget package from that branch?

**Comment by acottais** [comment]
I was going to ask the same thing. 
Just tell us what you think is left to be done to merge the branch I might be able to help.

**Comment by neilcampbell** [comment]
There is a TODO.txt file on that branch which has some of the known items. It would be awesome if you can grab any of those. "Run through a bunch of different failures scenarios to make sure the user can diagnose the problem." would be a great one for example. https://github.com/pact-foundation/pact-net/pull/175 has quite a bit of info on this, specifically https://github.com/pact-foundation/pact-net/pull/175#issuecomment-480868206

**Comment by neilcampbell** [comment]
Pretty much once all those items are ticked off, we should be good to merge.

**Comment by neilcampbell** [comment]
Two options for using the `message-pact` branch in it's current form:
1. You can build a nuget package by running the following commands from PowerShell:
- Create a release build from VS
- .\Build\Download-Standalone-Core.ps1 If you get a “The request was aborted: Could not create SSL/TLS secure channel.” error, also run [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
- .\Build\Publish-Release.ps1 -ReleaseVersionNumber "PACTNET_VERSION"
- .\Build\Publish-StandaloneCore-Releases.ps1 -ReleaseVersionNumber "PACTNET_VERSION"
Replace PACTNET_VERSION with the version number you want.
Once all the commands have been run you should see the packages in ./Build dir

2. https://ci.appveyor.com/project/SEEKJobs/pact-net/builds/25048346/artifacts is the latest build from message-pact

**Comment by mustodomenico** [comment]
Hello,

is there any documentation on how to use new available interface for async messages ?
Could you please point me to that ?
Thanks

**Comment by TheTFo** [comment]
Just checking in on this branch. Is there any active work happening on any of the TODO items?

**Comment by neilcampbell** [comment]
@TheTFo So far nothing has progressed just yet, I'll try make some time, but no promises as life has been pretty hectic.

**Comment by sfiodorov** [comment]
Is there any news on it?

**Comment by vpsouza** [comment]
Is there any news?

**Comment by DanisaurioRex** [comment]
Is there any news on it?

**Comment by steve-dickinson** [comment]
Has there been anymore progress on the message-pact? There is another project https://github.com/Mattersight/pact-net-messages which looked promising, but unfortunately there has been no more activity since September 2019.

**Comment by benquinteros** [comment]
bump
